platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/DependencySubstitutionRulesIntegrationTest.groovy

```diff
@@ -65,6 +63,7 @@ class DependencySubstitutionRulesIntegrationTest extends AbstractIntegrationSpec
                 failOnVersionConflict()
             }
         """
+        resolve.prepare()
```

Comment:
I haven't looked at what's generated....can this not be done in setup still?

Reply:
This writes text content to the root build script. 

Running `resolve.prepare` in setup prevents tests from using the plugins block.

We do this pattern in other tests.

We should find some time to rework the resolve test fixture so this is not necessary. 

I can go back and use `apply:`, though there is some precedent here